modules/gltf/SCsub

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env python
+
+Import("env")
+Import("env_modules")
+
+env_gltf = env_modules.Clone()
+env_gltf.Prepend(CPPPATH=["."])
```

Comment:
I'm not sure this is necessary, other modules don't see to need it to use file-relative includes?